### PR TITLE
Summary of available software updates

### PR DESCRIPTION
**Pull request by tomschr**
### PR creator: Description

Summary of available software updates and upgradable packages in host details view via SUMA integration

### PR creator: Are there any relevant issues/feature requests?

* change#1
* https://confluence.suse.com/x/_YA0WQ

### CODE REVIEW

**Comment by abravosuse** [comment]
@tomschr this PR is ready for your review.

File: trento/xml/article_sap_trento.xml [hunk 1]
```diff
@@ -1678,618 +1677,48 @@ In the Trento dashboard, go to the overview corresponding to the object for whic
...
-        can manage click the <guimenu>Checks catalog</guimenu>
-        entry in the left sidebar.
-      </para>
-      <figure>
-        <title><guimenu>Checks catalog</guimenu></title>
+      </figure>  
+   <para>Once the SUSE Manager settings are entered the SAP admin can test the 
+   connection by clicking on the Test Connection button. If the connection is successfull,
+   from that moment onward, for any registered host that is also managed by the SUSE Manager
+   instance which connection details we have provided, the corresponding Host Details view will
```
Comment (tomschr) [inline trento/xml/article_sap_trento.xml:1701]: The sentence starting with "If the connection is successful..." is quite long and hard to follow. Can we split it up into smaller chunks? Shouldn't the "is" be replaced with a "was"?

File: trento/xml/article_sap_trento.xml [hunk 2]
```diff
@@ -1678,618 +1677,48 @@ In the Trento dashboard, go to the overview corresponding to the object for whic
...
-        actions (what to do when the check is negative) and related
-        references. </para>
-      <para> What checks are carried out for each particular cluster is
-        determined using the <guimenu>Settings</guimenu> button in the
-        corresponding cluster detail view. See <xref
-          linkend="lst-trento-web-pace-cluster-settings" /> for
-        details. </para>
+   <para>There are three types of available patches: security advisories, bug fixes and feature
+   enhancement. Security advisory are regarded as critical and the existance of just one of them
+   will make the health of the host go critical. The others are regarding as warnings.</para>
```
Comment (tomschr) [inline trento/xml/article_sap_trento.xml:1714]: I would shorten it a bit. What about this:

```suggestion
   enhancement. Security advisories are regarded as critical. If one is found,
   the health of the host will be set to critical too. The others are to be seen as warnings.</para>
```

File: trento/xml/article_sap_trento.xml [hunk 3]
```diff
@@ -1678,618 +1677,48 @@ In the Trento dashboard, go to the overview corresponding to the object for whic
...
+    <para>At any given moment, the user can clear the SUSE Manager settings from the Settings view, which
+    will result in all the information regarding available software updates gone from the console and the
+    health of the hosts adjusted accordingly.</para>
```
Comment (tomschr) [inline trento/xml/article_sap_trento.xml:1717]: I would rephrase it a bit:

```suggestion
    <para>
    At any time, the user can clear the &suma; settings from the Settings view. 
    As a result, all information about available software updates disappears from the console and the status of the hosts is adjusted accordingly.</para>
```

File: trento/xml/article_sap_trento.xml [hunk 4]
```diff
@@ -1678,618 +1677,48 @@ In the Trento dashboard, go to the overview corresponding to the object for whic
...
+   <para>This integration with SUSE Manager is a first step in road which we expect will take Trento
+   to a point where the SAP admin is able  to orchestrate the patching of the Operating System of their
+   SAP environment using Trento and SUSE Manager behind it.</para>
```
Comment (tomschr) [inline trento/xml/article_sap_trento.xml:1720]: This sentence is about the future. I find this very problematic for the following reasons:

* It may be valid now, but who knows what will happen in 3 months or 1 year?
* Roadmaps can change.
* Technical documentation is about facts, not vague promises. :wink:  We document the current state, not any future impact.
* Could be a potential legal threat if a a customer want to hold us reliable for missing features.

As a conclusion, I would really recommend to remove it. It doesn't add any value or could potentially dangerous.

If you really think the user should know about future development, maybe link to the GitHub repo?

**Review by tomschr — COMMENTED** [review]
Thanks @abravosuse! See my comments below.

File: trento/xml/article_sap_trento.xml (hunk 1 above)
Comment (abravosuse) [reply trento/xml/article_sap_trento.xml]: My English is not that good. I defer the decision to an English native speaker.

File: trento/xml/article_sap_trento.xml (hunk 4 above)
Comment (abravosuse) [reply trento/xml/article_sap_trento.xml:1720]: Let's remove it then.

File: trento/xml/article_sap_trento.xml [hunk 5]
```diff
@@ -1678,618 +1683,45 @@ In the Trento dashboard, go to the overview corresponding to the object for whic
...
-        entry in the left sidebar.
-      </para>
-      <figure>
-        <title><guimenu>Checks catalog</guimenu></title>
+      </figure>  
+   <para>Once the &suma; settings are entered, the &sap; admin can test the 
+   connection by clicking on the <guimenu>Test Connection</guimenu> button. If the successfull,
+   from that moment onward, for any registered host that is also managed by the SUSE Manager
+   instance which connection details we have provided, the corresponding Host Details view will
+   display a summary of the available patches and the upgradable packages for that particular host:</para>
```
Comment (tomschr) [inline trento/xml/article_sap_trento.xml:1708]: @abravosuse I have a suggestion from one of my colleague: 

```suggestion
   <para>
   With the &suma; settings configured, the &sap; admin can test the connection
   by clicking <guimenu>Test Connection</guimenu>.
   If the connection is successful, the Host Details view displays a summary
   of the available patches and the upgradable packages for each host
   managed by &suma;:</para>
```

File: trento/xml/article_sap_trento.xml (hunk 1 above)
Comment (tomschr) [reply trento/xml/article_sap_trento.xml]: What about this:

```xml
With the &suma; settings configured, the &sap; admin can test the
connection by clicking <guimenu>Test Connection</guimenu>.

If the connection is successful, the Host Details view displays
a summary of the available patches and the upgradable packages
for each host managed by SUMA
```
